dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -357,7 +357,7 @@ class ApiAccessor {
         reportingPeriod: String,
         uploadFunction: (String, T, String) -> DataMetaInformation,
     ): DataMetaInformation {
-        val waitTime = 1000L
+        val waitTime = 1L
```

Comment:
Hier auch.

Reply:
Is correct as we now only need to wait 1ms instead of 1000 to ensure that we're not running into a Error 500